activemodel/lib/active_model/api.rb

```diff
@@ -67,20 +67,34 @@ module API
       extend ActiveModel::Translation
     end
 
-    # Initializes a new model with the given +params+.
+    ##
+    # :call-seq:
+    #   initialize(attributes, &block)
```

Comment:
@adrianna-chang-shopify the main reason I wanted to pass the `&block` as an argument was to emphasize the support in the RubyDoc. Is this the proper way to emphasize that while skipped the block initialization?